ui/pages/admin/adminOneBkAcc.html

```diff
@@ -0,0 +1,239 @@
+                                <td>4562 8654 8952 2458</td>
+                                <td>Claude</td>
+                                <td>For saving</td>
+                                <td>Credit</td>
+                                <td>200</td>
+                                <td>5000</td>
+                                <td>5200</td>
+                            </tr>
+                            <tr>
+                                <td>9:00 - 25th Jan, 2020</td>
+                                <td>4562 8654 8952 2458</td>
+                                <td>Claude</td>
+                                <td>For saving</td>
+                                <td>Credit</td>
+                                <td>200</td>
+                                <td>5000</td>
+                                <td>5200</td>
+                            </tr>
+                        </tbody>
+                    </table>
+
+
+                </div>
+
+
+            </div>
+        </div>
+
+        <div id="deleteBtn" class="deleteBtnMod">
+
+            <div class="openedMessage">
+                <span id="closeact" class="close">&times;</span>
+                <h3>Bank account deletion</h3>
+
+                <div class="cardPost ">
+                    <a href="# ">
+                        <div class="cardTexts ">
+                            <div class="cardTitle "> <a href="# ">John Doe</a></div>
+                            <div class="cardDesc "> <a href="# ">4568 5557 5754 7515</a></div>
+                            <div class="cardMeta ">
+                                <ul>
+                                    <li> <a>Status: <span>Active</span>  </a></li>
+                                    <li> <a>Type: <span>Savings</span>  </a></li>
+                                    <li> <a>Transactions: <span>10</span>  </a></li>
+
+                                </ul>
+                            </div>
+                        </div>
+                    </a>
+                </div>
+
+                <div class="message">This Bank account has been <span>Deleted</span> </div>
+
+            </div>
+
+        </div>
+
+
+        <script src="../../scripts/script.js"></script>
+        <script>
+            let delTran = document.getElementById("delTran");
+            let closeAct = document.getElementById("closeact");
+            let deleteBtn = document.getElementById("deleteBtn");
+            let status = document.getElementsByClassName('statu')[0]
+
+            delTran.onclick = () => {
+                deleteBtn.style.display = "block";
+                status.style.color = "#e42e0d"
+                status.innerHTML = "(Deleted)"
+            }
+            closeAct.onclick = () => deleteBtn.style.display = "none";
+
+            window.onclick = event => {
+                if (event.target == deleteBtn) {
+                    deleteBtn.style.display = "none";
+                }
+            }
```

Comment:
consider this comment: https://github.com/Irutingabo/Papel/pull/35#discussion_r378193958